<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrar Productos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblNombre">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipt_nombre">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipt_cantidad">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCantidad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrecio">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Precio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipt_precio">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="upSep">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="downSep">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_modificar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="Line" name="vertSep">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_consultar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>471</width>
      <height>192</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>471</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_borrar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
